### PR TITLE
chore: update examples

### PR DESCRIPTION
**Pull request by d3lm**
This PR updates the examples and slightly improves the `README` as well as adds a `package-lock.json`. By adding a lock file the package manager doesn't have to run resolution which makes booting the example almost instant.

### CODE REVIEW

**Comment by chenxsan** [comment]
I can see your point of improved performance, but I would suggest avoiding `package-lock.json` since it's impossible to review it.

File: examples/getting-started/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Getting Started
+
+## How to use
+
+Run the following commands in the root directory.
+
+```bash
+❯ npm install
+❯ npm run build
+❯ npm run serve
```
Comment (chenxsan) [inline examples/getting-started/README.md:10]:
```suggestion
$ npm install
$ npm run build
$ npm run serve
```

File: examples/module-federation/README.md [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+## How to use
 
-```
-yarn install
+Run the following commands in the root directory.
 
-yarn start (this should trigger the lerna command)
+```bash
+❯ npm install
+❯ npm start
```
Comment (chenxsan) [inline examples/module-federation/README.md:9]:
```suggestion
$ npm install
$ npm start
```

Comment (chenxsan) [reply examples/module-federation/README.md:9]: Another thing I found is this example won't work under npm 6 in my test. Only yarn works.

Comment (d3lm) [reply examples/module-federation/README.md:9]: Oh interesting. That's on local then right?

Comment (chenxsan) [reply examples/module-federation/README.md:9]: Right, I'm testing on my local computer.

File: examples/getting-started/README.md (hunk 1 above)
Comment (d3lm) [reply examples/getting-started/README.md]: Removed the sign completely.

**Review by chenxsan — APPROVED** [review]

**Pull request merged**